### PR TITLE
fix(DB/quest_poi): Update WorldMapAreaId to match respective quests

### LINKED ISSUE #8531: [Quest/POI] Some Barrens quests opens Durotar's map despite being only in The Barrens.
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour
There are some quests that opens Durotar's map despite being in the Barrens from the beginning to the end.

https://wowgaming.altervista.org/aowow/?quest=890
The quest "The Missing Shipment" opens Durotar's map.

https://wowgaming.altervista.org/aowow/?quest=892
The second part of the quest "The Missing Shipment" also opens Durotar's map.

https://wowgaming.altervista.org/aowow/?quest=896
The quest "Miner's Fortune" opens Durotar's map, no matter if the objective has been completed.

https://wowgaming.altervista.org/aowow/?quest=888
The quest "Stolen Booty", when the objective is not completed, opens Durotar.
Collecting only the "Telescopic Lens" (5077) lets you click the "Show Map" to open Durotar, but you cannot see where is the "Shipment of Boots" (5076) unless you navigate.
When the objective is completed, it shows Durotar.

https://wowgaming.altervista.org/aowow/?quest=887
The quest "Southsea Freebooters", when the objective is completed, doing "Show Map" opens Durotar's map instead of The Barrens.

I have also found, in addition to the original post, that the samophlange quests behave the same way (opening Durotar map). IDs 894, 900, 901, 902. The last quest (id 3924) opens The Barrens map though, so thats weird.

### Expected Blizzlike Behaviour
Clicking "Show Map" should opens the map of The Barrens.

### Source
No response

### Steps to reproduce the problem
Get the mentionned quests.
Click "Show Map".

### Extra Notes
No response

### AC rev. hash/commit
chromiecraft/azerothcore-wotlk@b2f0893

### Operating system
Ubuntu 20.04

### Modules
mod-ah-bot
mod-cfbg
mod-chromie-xp
mod-desertion-warnings
mod-duel-reset
mod-eluna-lua-engine
mod-ip-tracker
mod-low-level-arena
mod-multi-client-check
mod-pvp-titles
mod-pvpstats-announcer
mod-queue-list-cache
mod-server-auto-shutdown
lua-carbon-copy
lua-custom-worldboss
lua-level-up-reward
lua-recruit-a-friend
lua-send-and-bind
lua-temp-announcements
lua-zonecheck
Customizations
None

### Server
ChromieCraft

### Linked ChromieCraft Issue
https://github.com/chromiecraft/chromiecraft/issues/2100

### LINKED ISSUE #8713: POI of "Augustus' Receipt Book" shows Western Plaguelands instead of Eastern Plaguelands
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59

### Current Behaviour

Original report: https://github.com/chromiecraft/chromiecraft/issues/2175

Doing "Show Map" on the quest "Augustus' Receipt Book" (id 6164) opens Western Plaguelands map while both the objective and completion are in Eastern.

Tester:

The issue is confirmed. [Quest Augustus' Receipt Book](https://wowgaming.altervista.org/aowow/?quest=6164). "Show Quest Map" open Western Plaguelands map
WoWScrnShot_102421_001304

### Expected Blizzlike Behaviour

Doing "Show Map" on the quest "Augustus' Receipt Book" (id 6164) opens Eastern Plaguelands map.

### Source

[Youtube](https://youtu.be/jXcTdwgMnZI?t=9)

### Steps to reproduce the problem

.q a 6164

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b0d4f1917ec82d978121859b2ea45cdc178bb478

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### LINKED ISSUE #8770: POI of quests "The Spirits of Southwind" and "Hive in the Tower" shows Un'goro Crater instead of Silithus
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59

### Current Behaviour

Original report: https://github.com/chromiecraft/chromiecraft/issues/2200

Doing "Show Map" on the quest "The Spirits of Southwind" (1125) when completed opens the Un'Goro map.
The same happens with the quest "Hive in the Tower" (1126).

Tester:

The issues is confirmed. [The Spirits of Southwind](https://wowgaming.altervista.org/aowow/?quest=1125) showing always incorrect POI while [Hive in the Tower](https://wowgaming.altervista.org/aowow/?quest=1126) only with complete credit.

WoWScrnShot_102721_133917
WoWScrnShot_102721_134024

### Expected Blizzlike Behaviour

Because the hand-over NPC is in Silithus, doing "Show Map" should open Silithus's map.

### Source

X

### Steps to reproduce the problem

.q a 1125
click "Show quest Map"
.q a 1126
.q c 1126
click "Show quest Map"

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b0d4f1917ec82d978121859b2ea45cdc178bb478

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### LINKED ISSUE #9085: Quest Emberstrife incorrect POI
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59

### Current Behaviour

Quest [Emberstrife](https://wowgaming.altervista.org/aowow/?quest=6570) shows Thousand Needles map instead of Dustwallow Marsh

WoWScrnShot_110921_153112

### Expected Blizzlike Behaviour

Quest end in Dustwallow Marsh so it should open it map

### Source

X

### Steps to reproduce the problem

.q a 6570
show quest map

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/12e92b58082dfda8b73b50aa97b98b9479bb7629

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by Voxstrasza**
## Changes Proposed:
-  Update WorldMapAreaId to match respective quests in issues linked below

## Issues Addressed:
- Closes #8531
- Closes #8713
- Closes #8770
- Closes #9085
- Closes chromiecraft/chromiecraft#2100
- Closes chromiecraft/chromiecraft#2175
- Closes chromiecraft/chromiecraft#2200

## How to Test the Changes:
1. Add the quests 894, 900, 901, 902, 890, 892, 896, 888, 887, 3924, 6164, 1125, 1126 and 6570 to your quest log
2. Right click on quest tracker and show map
3. Should now bring you to where the quest is located and not the next zone over

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1638942051959578400.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=887 AND `id`=3;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=894 AND `id`=0;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=900 AND `id`=0;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=900 AND `id`=1;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=900 AND `id`=2;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=900 AND `id`=3;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=901 AND `id`=0;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=901 AND `id`=1;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=902 AND `id`=0;
+UPDATE `quest_poi` SET `WorldMapAreaId`='11' WHERE `QuestID`=3924 AND `id`=3;
```
Comment (IntelligentQuantum) [inline data/sql/updates/pending_db_world/rev_1638942051959578400.sql:30]: `WorldMapAreaId`='11' > `WorldMapAreaId`=11
`WorldMapAreaId`='23' > `WorldMapAreaId`=23

**Review by IntelligentQuantum — CHANGES_REQUESTED** [review]

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by acidmanifesto** [comment]
@azerothcore/testers please confirm this sql based pr is tested working.

**Comment by Shineslippers** [comment]
Tested:

- [X] Quest 887. Both `accept` and `complete` POI are correct
- [X] Quest 888. Both `accept` and `complete` POI are correct
- [X] Quest 890. Both `accept` and `complete` POI are correct
- [X] Quest 892. Both `accept` and `complete` POI are correct
- [X] Quest 896. Both `accept` and `complete` POI are correct
- [X] Quest 896. Both `accept` and `complete` POI are correct
- [X] Quest 1125. Both `accept` and `complete` POI are correct
- [X] Quest 1126. Both `accept` and `complete` POI are correct
- [X] Quest 6164. Both `accept` and `complete` POI are correct
- [X] Quest 6570. Both `accept` and `complete` POI are correct

<details><summary>Spoiler</summary>

WoWScrnShot_120821_230643
WoWScrnShot_120821_230857
WoWScrnShot_120821_230953
WoWScrnShot_120821_231002
WoWScrnShot_120821_231016
WoWScrnShot_120821_231025
WoWScrnShot_120821_231118
WoWScrnShot_120821_231241
WoWScrnShot_120821_231343
WoWScrnShot_120821_231417
WoWScrnShot_120821_231425
WoWScrnShot_120821_231459
WoWScrnShot_120821_231511
WoWScrnShot_120821_231543
WoWScrnShot_120821_231551


</details>

**Comment by Voxstrasza** [comment]
> Quest 3924. `accept` POI is correct, but `complete` doesn't open Orgrimmar map.

okay, I'll fix that

**Comment by Shineslippers** [comment]
@Voxstrasza please use all links from AC reports to your `Issues Addressed` field. There is existing `Original report` line in most of reports. Thank you.

**Comment by Voxstrasza** [comment]
Quest 3924 will now open map Orgrimmar when completion requirements are met.  https://i.imgur.com/7Bri9Dh.jpg

**Review by acidmanifesto — APPROVED** [review]

**Pull request merged**